src/containers/WelcomePage/Components/Programmes.tsx

```diff
@@ -30,14 +36,16 @@ const StyledWrapper = styled.div`
   }
 `;
 
-const Desktop = styled.div`
+const Desktop = styled.ul`
   display: none;
   ${mq.range({ from: breakpoints.tablet })} {
     display: flex;
     justify-content: space-evenly;
     flex-wrap: wrap;
     gap: ${spacing.normal};
   }
+
+  ${UlStyling};
```

Comment:
Hvis dette bare brukes et sted ville jeg nok bare inlinet det.

Reply:
Brukes også i ul for mobilvisning. Kan inline det der siden det er usannsynlig å forandre det i framtiden 🥇

Reply:
Den er grei!